modules/appeals_api/app/controllers/concerns/appeals_api/icn_parameter_validation.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module IcnParameterValidation
```

Comment:
I put the logic about ICN parameter validation into this concern, since the Legacy Appeals API and Appealable Issues API will be using this same behavior. I have a branch with the required changes for both those APIs, but I couldn't fit it under the line limit for this PR.